Sources/SourceKitLSP/Swift/FoldingRange.swift

```diff
@@ -164,6 +164,13 @@ fileprivate final class FoldingRangeFinder: SyntaxAnyVisitor {
     return self.addFoldingRange(start: start, end: end)
   }
 
+  override func visit(_ node: FunctionParameterClauseSyntax) -> SyntaxVisitorContinueKind {
```

Comment:
Should we just support all `ParenthesizedSyntax` 🤔?

Reply:
Good idea 👍🏽